decidim-assemblies/app/queries/decidim/assemblies/private_assemblies.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    # This query class filters assemblies given a current_user.
+    class PrivateAssemblies < Rectify::Query
```

Comment:
I've struggled a bit to understand what this class actually does. It looks like instead of returning `PrivateAssemblies`, it actually returns `VisibleAssemblies` right? If a user is provided, it will scope them to all the assemblies the user can see. Otherwise, it will return all the public assemblies.